### ISSUE TITLE
Homepage - Links in deck area

### ISSUE THREAD

**Issue by sbxchicago** [betterscientificsoftware/betterscientificsoftware.github.io#103]
Can you remove the bulleted links in the deck area of homepage? Clara is going to programmatically assign the correct links so that they can be distributed to each of the subpages

**Comment by sbxchicago** [comment]
Let me rephrase this, can you remove these items altogether? Then Clara will fix the programmatic links.

**Comment by curfman** [comment]
Done, commit 1d385d824378ed9ccc3b58154e7279fd7b857a29
